contracts/Delegations.sol

```diff
@@ -131,15 +131,66 @@ contract Delegations is IDelegations, IStakeChangeNotifier, ManagedContract {
 		_;
 	}
 
-	function importDelegations(address[] calldata from, address[] calldata to, bool refreshStakeNotification) external onlyMigrationManager onlyDuringDelegationImport {
-		require(from.length == to.length, "from and to arrays must be of same length");
+	function importDelegations(address[] calldata from, address to, bool refreshStakeNotification) external onlyMigrationManager onlyDuringDelegationImport {
 
+		uint256 uncappedStakesDelta = 0;
```

Comment:
either:
require to != address(0)
or 
require to != address(0) or from.length == 1 and set to = from[0] if to == address(0)

Reply:
adding:
```
		require(to != address(0), "to must be a non zero address");
		require(from.length > 0, "from array must contain at least one address");
```